### PR TITLE
Spanish traslation

### PR DESCRIPTION
**Pull request by wero1414**
Hello!

We have been working and testing the pages, please review it

Thanks!

### CODE REVIEW

File: content/credits/_index.es.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* [Mike Thompson](https://www.linkedin.com/in/michael-thompson-0a581a/) por su experticia en verificación
+* [Jix](https://twitter.com/jix_) por soporte de verificación formal
+* [Proppy](https://twitter.com/proppy) por ayudar con Github actions
+* [Maximo Balestrini](https://twitter.com/maxiborga) por todos los increíbles renders y el visor GDS interactivo
+* James Rosenthal por presentar ejemplos con diseño digital
+* Todas las personas que fueron parte de [TinyTapeout 01](/runs/tt01) y ofrecieron su tiempo de manera voluntaria para mejorar la documentación y probar el flujo.
+* El equipo de [YosysHQ](https://www.yosyshq.com/) y todos los otros desarrolladores de herramientas EDA de código abierto
+* [Efabless](https://efabless.com/) por realizar los lanzamientos y proporcionar OpenLane y patrocinio
+* [Tim Ansell y Google](https://www.youtube.com/watch?v=EczW2IWdnOM) por apoyar al movimiento silicio de código abierto
+* Comunidad del curso [Zero to ASIC](https://zerotoasiccourse.com/) por todo tu apoyo
```
Comment (mattvenn) [inline content/credits/_index.es.md:25]: please add yourselves for the spanish translation!

File: content/digital_design/design_uart/_index.en.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ This circuit, inspired by @maehw, implements a shift register with 17 bits: four
 
 The circuit supports transmitting a user-selected ASCII character from 0x40 (@) to 0x5F (_), including capital letters from the Latin alphabet. 
 
-![ASCIItable](images/asciitable.png)
+![ASCIItable](/images/design_uart/asciitable.png)
```
Comment (mattvenn) [inline content/digital_design/design_uart/_index.en.md:34]: this breaks the image for me

**Review by mattvenn — COMMENTED** [review]
why move all the images?

File: content/credits/_index.es.md (hunk 1 above)
Comment (wero1414) [reply content/credits/_index.es.md:25]: shure!

**Pull request merged**